app/src/main/java/org/mozilla/fenix/settings/PairFragment.kt

```diff
@@ -35,13 +39,17 @@ class PairFragment : Fragment(R.layout.fragment_pair), BackHandler {
                 onNeedToRequestPermissions = { permissions ->
                     requestPermissions(permissions, REQUEST_CODE_CAMERA_PERMISSIONS)
                 },
-                onScanResult = { pairingUrl ->
-                    requireComponents.services.accountsAuthFeature.beginPairingAuthentication(
-                        requireContext(),
-                        pairingUrl
-                    )
-                    findNavController(this@PairFragment)
-                        .popBackStack(R.id.turnOnSyncFragment, false)
+                onScanResult = {
+                        pairingUrl ->requireComponents.services.accountsAuthFeature.beginPairingAuthentication(requireContext(),pairingUrl)
+                    val vibrator = context?.getSystemService(Context.VIBRATOR_SERVICE) as Vibrator
+                    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+                        vibrator.vibrate(VibrationEffect.createOneShot(200, VibrationEffect.DEFAULT_AMPLITUDE))
```

Comment:
`200` should be a constant, especially since it gets used in two places.

```kt
companion object {
  private const val VIBRATE_LENGTH = 200L
}
```

Reply:
Sure @NotWoods